divHretention/plotting/plot_along_divertor.py

```diff
@@ -250,3 +250,25 @@ def __init__(self, **kwargs):
         super().__init__(
             quantities=quantities,
             **kwargs)
+
+
+def plot_inv_with_uncertainty(x, y, stdev, zscore=2, alpha_fill=0.3, **kwargs):
+    """Plots the inventory along the divertor with the associated uncertainity
+    filled
+
+    Args:
+        x (numpy.array): Arc length (m) along the divertor
+        y (numpy.array): Inventory per unit thickness (H/m) 
+        stdev (numpy.array): standard deviation
+        zscore (float): userdefined zscore corresponding to a confidence interval. Defaults to 2.
+        alpha_fill (float, optional): Shows the boundaries of uncertainty. Defaults to 0.3.
```

Comment:
No, this parameter is the opacity.
```suggestion
        alpha_fill (float, optional): Opacity of the filled region between 0 and 1. Defaults to 0.3.
```